service/src/test/java/tds/assessment/services/impl/AssessmentServiceImplTest.java

```diff
@@ -277,4 +292,139 @@ public void shouldReturnAssessmentWithItemProps() {
         assertThat(retCon2.getPropertyName()).isEqualTo(constraint2.getPropertyName());
         assertThat(retCon2.getPropertyValue()).isEqualTo(constraint2.getPropertyValue());
     }
+
+    @Test
+    public void shouldReturnAssessmentWithStrands() {
```

Comment:
Moved this test into AssessmentAssemblerTest - testing the assembly of the entire assessment in one method now.